### PR TITLE
Add tsg docs

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: docs/howto-tsg.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# How to troubleshooting issues
```
Comment (chenkennt) [inline docs/howto-tsg.md]: connectivity and message delivery issues?

Comment (chenkennt) [reply docs/howto-tsg.md]: How to *troubleshoot*

File: docs/howto-tsg.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+- [Default mode troubleshooting](#default_mode_tsg)
+    - [How to view traffic](#view_traffic)
+        - [Client requests](#view_traffic_client)
+        - [Server requests](#view_traffic_server)
+    - [How to add logs](#add_logs)
+        - [Client-side logs](#add_logs_client)
+        - [Server-side logs](#add_logs_server)
+            - [For ASP.NET Core SignalR](#add_logs_server_aspnetcore)
+            - [For ASP.NET SignalR](#add_logs_server_aspnet)
+#### How to enable server-side log
```
Comment (chenkennt) [inline docs/howto-tsg.md]: Why this is a h4?

File: docs/howto-tsg.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      <add name="SignalRSwitch" value="Information" />
+    </switches>
+    <!-- Specifies the trace writer for output -->
+    <sharedListeners>
+      <add name="ASRS" type="System.Diagnostics.TextWriterTraceListener" initializeData="asrs.log.txt" />
+    </sharedListeners>
+    <trace autoflush="true" />
+  </system.diagnostics>
+```
+
```
Comment (chenkennt) [inline docs/howto-tsg.md]: Also customer can enable service-side logs to help troubleshooting

File: docs/howto-tsg.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    <switches>
+      <add name="SignalRSwitch" value="Information" />
+    </switches>
+    <!-- Specifies the trace writer for output -->
+    <sharedListeners>
+      <add name="ASRS" type="System.Diagnostics.TextWriterTraceListener" initializeData="asrs.log.txt" />
+    </sharedListeners>
+    <trace autoflush="true" />
+  </system.diagnostics>
+```
```
Comment (chenkennt) [inline docs/howto-tsg.md]: Any guidance if customer sees exception after enabling the log?

File: docs/howto-tsg.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+        - [Server-side logs](#add_logs_server)
+            - [For ASP.NET Core SignalR](#add_logs_server_aspnetcore)
+            - [For ASP.NET SignalR](#add_logs_server_aspnet)
+#### How to enable server-side log
+- [Serverless mode troubleshooting](#serverless_mode_tsg)
+- [Classic mode troubleshooting](#classic_mode_tsg)
+
+We encountered several times when users say "service is not working", however we need more information in detail to find out the root cause of the "not working" issue. This guidance is to introduce several ways to help do self-diagnosis to find the root cause directly or narrow down the issue and report it to us for further investigation.
+
+First, please check from the Azure portal which ![ServiceMode](./images/service-mode.png) is the Azure SignalR Service (aka. **ASRS**) configured to.
```
Comment (JialinXin) [inline docs/howto-tsg.md]: Image in middle of content make it hard to read. Better to move image to a different line.

**Pull request merged**